src/styles/_catalog.scss

```
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(1, $cardWidth);
```

You shouldn't use repeat for only one column. Just keep $cardWidth

```suggestion
  grid-template-columns: $cardWidth;
```

---

agree